Retrospective/week_5_(group3).md

```diff
@@ -0,0 +1,17 @@
+# Retrospective for Week 5 (Group 3- Fork The Data)
+
+## To-Do - On going development
+- Need to integrate Interest profiler with scrapped dataset and have a sample output ready before presentation
+- Make Progress in Recommeder system 
+	- Data correction is to be done for column "CAO points" in m_dataset. (more accurate data scrapping is required if needed)
+	- For Collaborative filtering-based systems with LSTM, we need to have it display more than 5 fields in the prefernce output. (CNN is not the most accurate)
+	- Content-based Recommender Systems will be main point of focus for week 5,6,7 as we will be improving on the accuray rate of the output.
+- UI is not scalable as of now, We need to make it dynamically scalable on all the devices by end of next week.
+- Existing Sentiment analysis notebook is being extended to work with the scraped reviews (r_dataset) of TUD campuses.
+
+## Important Points to be taken into account
+- Delete old source code of the website which is not in use from github
```

Comment:
Old source code of the website has already been removed. Responsible frontend code is under frontend folder and responsible backend code is under backend folder.